lib/settings/pages/user_labels_settings_page.dart

```diff
@@ -0,0 +1,203 @@
+import 'dart:async';
+
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:smooth_highlight/smooth_highlight.dart';
+import 'package:thunder/account/models/user_label.dart';
+
+import 'package:thunder/core/enums/local_settings.dart';
+import 'package:thunder/post/utils/user_label_utils.dart';
+import 'package:thunder/shared/dialogs.dart';
+import 'package:thunder/shared/full_name_widgets.dart';
+import 'package:thunder/shared/input_dialogs.dart';
+
+class UserLabelSettingsPage extends StatefulWidget {
+  final LocalSettings? settingToHighlight;
+
+  const UserLabelSettingsPage({super.key, this.settingToHighlight});
+
+  @override
+  State<UserLabelSettingsPage> createState() => _UserLabelSettingsPageState();
+}
+
+class _UserLabelSettingsPageState extends State<UserLabelSettingsPage> with SingleTickerProviderStateMixin {
+  GlobalKey settingToHighlightKey = GlobalKey();
+  LocalSettings? settingToHighlight;
+
+  List<UserLabel> userLabels = [];
+
+  void _updateChangedUserLabel(({UserLabel? userLabel, bool deleted}) result) {
+    if (result.userLabel != null) {
```

Comment:
Invert the conditional statement to reduce nesting!

```suggestion
    if (result.userLabel == null) return;
```

Reply:
Fixed this!